src/client/app/components/csv/ReadingsCSVUploadComponent.tsx

```diff
@@ -3,338 +3,490 @@
+								</Input>
+							</Label>
+						</FormGroup>
+						<FormGroup>
+							<Container>
+								<Row>
+									<Col>
+										<Label for='gzip'>
+											<div style={checkBox}>
+												<Input
+													type='checkbox'
+													id='gzip'
+													name='gzip'
+													onChange={handleCheckboxChange}
+												/>
+												<div className='ps-2'>
+													{translate('csv.common.param.gzip')}
+												</div>
+											</div>
+										</Label>
+									</Col>
+									<Col>
+										<Label for='headerRow'>
+											<div style={checkBox}>
+												<Input
+													type='checkbox'
+													id='headerRow'
+													name='headerRow'
+													onChange={handleCheckboxChange}
+												/>
+												<div className='ps-2'>
+													{translate('csv.common.param.header.row')}
+												</div>
+											</div>
+										</Label>
+									</Col>
+								</Row>
+								<Row>
+									<Col>
+										<Label for='update'>
+											<div style={checkBox}>
+												<Input
+													type='checkbox'
+													id='update'
+													name='update'
+													onChange={handleCheckboxChange}
+												/>
+												<div className='ps-2'>
+													{translate('csv.common.param.update')}
+												</div>
+											</div>
+										</Label>
+									</Col>
+									<Col>
+										<Label for='relaxedParsing'>
+											<div style={checkBox}>
+												<Input
+													type='checkbox'
+													id='relaxedParsing'
+													name='relaxedParsing'
+													onChange={handleCheckboxChange}
+												/>
+												<div className='ps-2'>
+													{translate('csv.readings.param.relaxed.parsing')}
+												</div>
+											</div>
+										</Label>
+									</Col>
+								</Row>
+								<Row>
+									<Col>
+										<Label for='honorDst'>
+											<div style={checkBox}>
+												<Input
+													type='checkbox'
+													id='honorDst'
+													name='honorDst'
+													onChange={handleCheckboxChange}
+												/>
+												<div className='ps-2'>
+													{translate('csv.readings.param.honor.dst')}
+												</div>
+											</div>
+										</Label>
+									</Col>
+									<Col>
+										<Label for='refreshReadings'>
+											<div style={checkBox}>
+												<Input
+													type='checkbox'
+													id='refreshReadings'
+													name='refreshReadings'
+													onChange={handleCheckboxChange}
+												/>
+												<div className='ps-2'>
+													{translate('csv.readings.param.refresh.readings')}
+												</div>
+											</div>
+										</Label>
+									</Col>
+								</Row>
+							</Container>
+						</FormGroup>
+						<div className='d-flex flex-row-reverse'>
+							<div className='p-3'>
+								<Button color='primary' type='submit' disabled={!isValidCSV}>
```

Comment:
I like the disabled check. I also see the warning if you don't select a meter and submit. Would it make sense to not allow submit on no meter selected?